ssr/src/page/post_view/bet.rs

```diff
@@ -177,6 +174,7 @@ fn HNButtonOverlay(
         }
     });
 
+    let tracked_coin = move || coin.with_untracked(|v| u64::from(*v));
```

Comment:
this isnt tracked, perhaps you mean current_coin?

Reply:
yes , coin value as updated from up and down arrow